github/search.go

```diff
@@ -24,6 +24,10 @@ import (
 // will search for such issues, sorting by creation date in ascending order
 // (i.e., oldest first).
 //
+// If query includes multiple condition, it MUST NOT include "+" as condition separator.
```

Comment:
nit: Could you please tweak this slightly so that it is easier to read?  How about:

```
// If query includes multiple conditions, it MUST NOT include "+" as the condition separator.
```

Reply:
Sure, i will add more detail (using " " instead) to the documentation. Thanks for your review.